nixos/tests/lorri/default.nix

```diff
@@ -0,0 +1,26 @@
+import ../make-test-python.nix {
+  machine = { pkgs, lib, ... }: {
+    imports = [ ../../modules/profiles/minimal.nix ];
+    environment.systemPackages = [ pkgs.lorri ];
+  };
+
+  testScript = ''
+    # Copy files over
+    machine.execute(
+        """echo '${builtins.readFile ./fake-shell.nix}' > shell.nix"""
```

Comment:
please just use single double quotes here… or did `black` suggest to do that?

Reply:
`fake-shell.nix` contains single `"` quotes and is inserted into this file using `readFile`, so I need to keep the triple quotes to make sure the resulting Python is syntactically correct.

Reply:
Urgh, true. But you can probably just cp `${./fake-shell.nix} shell.nix`, can't you?

Reply:
Yes! I don't have a good mental model for how the test machines are instantiated and precisely what they have access to, but `cp` works. Cheers.